charts/kubeclarity/values.yaml

```diff
@@ -41,6 +41,31 @@ kubeclarity:
 
   service:
     type: ClusterIP
+    annotations: {}
+
+  ingress:
+    # Be careful when using ingress. As there is no authentication on Kubeclarity yet, Your instance may be accessible. 
+    # Make sure the ingress remains internal if you decide to enable it.
+    enabled: false
+    labels: {}
+    annotations: {}
+
+    # Optionally use ingressClassName instead of deprecated annotation.
+    # See: https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation
+    ingressClassName: ""
+
+    hosts:
+        # hostname you want to use
+      - host: chart-example.local
+        # path will default to /
```

Comment:
nit: a slight better comment about the default:
```
# paths will default to:
# - paths:
#    - pathType: Prefix
#      path: /
paths: []
```

Reply:
Updated